src/tests/unit/audit_remediation/test_audit_remediation_base.py

```diff
@@ -334,6 +345,8 @@ def test_result_query_exports_fail_without_run_id(cb):
         result_query.export_csv_as_string()
     with pytest.raises(ApiError):
         result_query.export_csv_as_file('blort.txt')
+    with pytest.raises(ApiError):
+        list(result_query.export_csv_as_lines())
```

Comment:
You didn't have to tell me what I needed to do here to make this work, @avanbrunt-cb.  I figured it out on my own. :)